lib/Cake/Controller/Component/Auth/BasicAuthenticate.php

```diff
@@ -82,8 +82,17 @@ public function authenticate(CakeRequest $request, CakeResponse $response) {
  * @return mixed Either false or an array of user information
  */
 	public function getUser(CakeRequest $request) {
-		$username = env('PHP_AUTH_USER');
-		$pass = env('PHP_AUTH_PW');
+		$username = '';
+		$pass = '';
+		if (isset($_SERVER['PHP_AUTH_USER'])) {
```

Comment:
any news on this PR? I just submitted a bugfix as I used the code in a brand new cakephp project con FastCGI and it's working flawlessly.